.cirrus.yml

```diff
@@ -68,6 +68,18 @@ task:
     - cd Tests/Packet++Test && Bin/Packet++Test
     - cd ../Pcap++Test && Bin/Pcap++Test -n
 
+task:
+  name: Test Apple Silicon (arm64) MacOS
+  osx_instance:
+    image: catalina-xcode
+  configure_script:
+    - ./configure-mac_os_x.sh --arm64
+  build_script:
+    - make all
+  check_for_arm64_slice_script:
+    - cd Dist && ALL_EXECUTABLES_WITHOUT_SYMLINKS=($(find . -perm +111 -type f)) && for EXECUTABLE in ${ALL_EXECUTABLES_WITHOUT_SYMLINKS[@]}; do echo "Checking for arm64 in $EXECUTABLE"; CHECK_ARCH=$(lipo $EXECUTABLE -verify_arch arm64 2>&1); RETVAL=$?; if [ $RETVAL -eq 1 ]; then echo "Couldn't find arm64 slice in $EXECUTABLE"; exit 1; else echo "Found arm64 slice in $EXECUTABLE"; fi; done
+    - cd ../Dist && ALL_LIBRARIES_WITHOUT_SYMLINKS=($(find . -name "*.a" -type f)) &&  for LIBRARY in ${ALL_LIBRARIES_WITHOUT_SYMLINKS[@]}; do echo "Checking for arm64 in $LIBRARY"; CHECK_ARCH=$(lipo $LIBRARY -verify_arch arm64 2>&1); RETVAL=$?; if [ $RETVAL -eq 1 ]; then echo "Couldn't find arm64 slice in $LIBRARY"; exit 1; else echo "Found arm64 slice in $LIBRARY"; fi; done
```

Comment:
I think we can remove that part? 🤔

Reply:
@seladb  : Do you mean to remove the whole `.cirrus.yml` update or to remove only the arm64 slice check?

Reply:
I think we can remove the `.cirrus.yml` task if we have a similar task that runs in GitHub Actions, right?

Reply:
Yes, sure.